projects/gdpr-compliance/README.md

```diff
@@ -0,0 +1,14 @@
+## Make OSM comply with the GDPR
+
+The OSM website and API need to comply with the GDPR in relation to the privacy of users and user accounts.
+
+* Project: [openstreetmap-website](https://github.com/openstreetmap/openstreetmap-website)
+* Related:
+  - https://blog.openstreetmap.org/2018/05/14/preparing-for-the-gdpr/
+  - https://2018.stateofthemap.org/slides/W016-The_LWG_Presents__GDPR_Implementation_for_OSM.pdf
+
+The expected deliverable is a mergeable pull request towards https://github.com/openstreetmap/openstreetmap-website. It must implement the changes to the website and API related to the privacy of user accounts in  the [GDPR Position Paper](https://wiki.openstreetmap.org/w/images/8/88/GDPR_Position_Paper.pdf) and [list of affected services](https://wiki.openstreetmap.org/wiki/GDPR/Affected_Services).

```

Comment:
This will likely take at least 20 pull requests, **a** pull request is not going to happen. I already made five.